dali/operators/reader/loader/video_loader.h

```diff
@@ -95,6 +96,13 @@ struct OpenFile {
   AVCodecContext* codec;
 #endif
   av_unique_ptr<AVFormatContext> fmt_ctx_;
+
+  struct VideoFileData {
+    FILE *file_stream;
+    long int file_position;
+    std::string filename;
+  };
+  VideoFileData file_desc;
```

Comment:
Nitpick: why not `VideoFileDesc` or `file_data`?

Reply:
Done